packages/runtime/datastore-definitions/api-report/datastore-definitions.api.md

```diff
@@ -112,7 +112,7 @@ export interface IFluidDataStoreRuntime extends IEventProvider<IFluidDataStoreRu
     // (undocumented)
     readonly IFluidHandleContext: IFluidHandleContext;
     // (undocumented)
-    readonly logger: ITelemetryLogger;
+    readonly logger: ITelemetryLoggerExt;
```

Comment:
This has to be `Base` because it's on the public type

Reply:
(oops already covered below, sorry)

Reply:
Ah! Resolved.